packages/swirl-components/src/components/swirl-tree-navigation-item/swirl-tree-navigation-item.css

```diff
@@ -1,78 +1,127 @@
 :host {
   display: inline-flex;
   width: 100%;
+}
 
-  & * {
-    box-sizing: border-box;
-  }
+:host * {
+  box-sizing: border-box;
 }
 
+/* Base item styles */
 .tree-navigation-item {
```

Comment:
Issue: With a long label the item is extending past the visible viewport and gets clipped. There seems to be something off with the truncation.